packager/media/formats/mp2t/adts_header.cc

```diff
@@ -89,6 +88,13 @@ size_t AdtsHeader::GetFrameSize() const {
   return frame_size_;
 }
 
+size_t AdtsHeader::GetFrameSizeWithoutParsing(const uint8_t* data,
+                                              size_t num_bytes) const {
+  DCHECK_GT(num_bytes, 5);
```

Comment:
Ah, there are still failures on Windows: https://travis-ci.org/google/shaka-packager/builds/591706925?utm_source=github_status&utm_medium=notification

You can probably fix it by either

```c++
DCHECK_GT(num_bytes, 5u);
```

or 

```C++
DCHECK_GT(num_bytes, static_cast<size_t>(5));
```